nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/QueryCassandra.java

```diff
@@ -379,7 +406,7 @@ public static long convertToAvroStream(final ResultSet rs, final OutputStream ou
      * @throws TimeoutException     If a result set fetch has taken longer than the specified timeout
      * @throws ExecutionException   If any error occurs during the result set fetch
      */
-    public static long convertToJsonStream(final ResultSet rs, final OutputStream outStream,
+    public static long convertToJsonStream(final Optional<ProcessContext> context, final ResultSet rs, final OutputStream outStream,
```

Comment:
In my opinion we should not expose this method as `public`. It is unnecessary in general and I believe the signature will change when the SimpleDateFormat code is migrated to `java.time`.
The javadoc should be moved to the original method (otherwise the new parameter should be added here).

Reply:
Thanks @turcsanyip , I've updated the code according to your comment.